src/modules/TeamsList/components/Teams/components/MyTeam/components/MyTeamInfoBlock/MyTeamInfoBlock.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { FC } from 'react';
+import { StyledMyTeamInfoBlock, InfoButton, EditButton } from './styled';
+import { MyTeamInfoLine } from './components/MyTeamInfoLine/MyTeamInfoLine';
```

Comment:
Name file index.tsx and change import to import { MyTeamInfoLine } from './components/MyTeamInfoLine';

Reply:
done